src/backends/interflop-ieee/Makefile.am

```diff
@@ -1,6 +1,6 @@
 lib_LTLIBRARIES = libinterflop_ieee.la
 libinterflop_ieee_la_SOURCES = interflop_ieee.c ../../common/printf_specifier.c ../../common/logger.c
-libinterflop_ieee_la_CFLAGS = -DBACKEND_HEADER="interflop_ieee"
+libinterflop_ieee_la_CFLAGS = -DBACKEND_HEADER="interflop_ieee" -O3
```

Comment:
same as before, just to remeber there is more than one

Reply:
There is no real improvements for the IEEE backend.